tslint.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
yeah, the default ones for tslint are nice, there are ones that are a bit of a PITA but are nice for standardization, like ordering imports alphabetically.